README.md

```diff
@@ -1,96 +1,127 @@
 # About
-This addon is an extension of the [SMPL-X Blender addon](https://www.youtube.com/watch?v=DY2k29Jef94) created by Joachim Tesch and owned by the Max Planck Institute for Intelligent Systems. Meshcapade has licensed the SMPL model and its updates (including SMPL+H, SMPL-X and STAR) from Max Planck Institute, and is making this extension of the blender add-on available for commercial users who have an active license for the SMPL Model. 
+This addon is an extension of the [SMPL-X Blender addon](https://www.youtube.com/watch?v=DY2k29Jef94) created by Joachim Tesch and owned by the Max Planck Institute for Intelligent Systems. Meshcapade has licensed the SMPL model and its updates (including SMPL-H, SMPL-X, STAR and SUPR) from Max Planck Institute, and is making this extension of the blender add-on available for commercial users who have an active license for the SMPL Model. 
 
-For academic uses, please visit: https://smpl-x.is.tue.mpg.de/. Or contact us at support@meshcapade.com
+Academic use of this plugin is free - just contact us at support@meshcapade.com.
 
-# SMPL-X and Meshcape Utilities Blender Add-on
+This add-on allows you to add [SMPL-H](https://mano.is.tue.mpg.de/), [SMPL-X](https://smpl-x.is.tue.mpg.de), and [SUPR](https://supr.is.tue.mpg.de) bodies to your current Blender scene. Each body consists of a mesh, a shape specific rig, and shape keys (blend shapes) for shape, expression and pose correctives. This addon was most recently developed in Blender 3.5.1.
```

Comment:
I think what's going on is that in line 2 we say we're licensing a bunch of stuff from MPI including STAR, but since the blender-worker doesn't output STAR, the blender plugin doesn't support it.

I developed it in 3.5.1.  During development 3.6 came out, and I didn't update during development.  Maybe I should just remove this statement.